flarepredict_train.py

```diff
@@ -48,20 +48,20 @@ def train_model(args):
 '''
 Command line parameters parser
 '''
-parser = argparse.ArgumentParser()
-parser.add_argument('-t', '--train_data_file', default=None,
-                    help='full path to a file includes training data to create a model, must be in csv with comma separator')
-parser.add_argument('-f', '--flare', default='C',
-                    help='Flare category to use for training. Available algorithms: C, M, and M5')
-parser.add_argument('-m', '--modelid', default='default_model',
-                    help='model id to save or load it as a file name. This is to identity each trained model.')
+# parser = argparse.ArgumentParser()
+# parser.add_argument('-t', '--train_data_file', default=None,
+#                     help='full path to a file includes training data to create a model, must be in csv with comma separator')
+# parser.add_argument('-f', '--flare', default='C',
+#                     help='Flare category to use for training. Available algorithms: C, M, and M5')
+# parser.add_argument('-m', '--modelid', default='default_model',
```

Comment:
if command line args are not used, please remove them entirely , not just comment out.